intranet/static/js/eighth/schedule.js

```diff
@@ -175,5 +175,21 @@ $(function() {
         if (activities !== "" && !confirm("Are you sure you want to add the following activities without a sponsor?\n" + activities)) {
             e.preventDefault();
         }
+        var stickyCancel = "";
+
+        $("tr.form-row:not(.hidden)").each(function(i, el) {
+          if($(this).hasClass("scheduled")){
+
+              var scheduledField = $("td[data-field='scheduled'] input", el);
+              var stickyField = $("td[data-field='sticky'] input", el);
+              if (!scheduledField.prop("checked")) {
+                  if(stickyField.prop("checked") || activity_sticky) {
+                      stickyCancel += "\n    " + $(".block-name a.ui-link", el).text().trim();
+                  }
+              }
+          }
+      });
+      if (stickyCancel !== "" && !confirm("Cancelling the following sticky activities will allow students to switch out of them. Are you sure?\n" + stickyCancel)){
+          e.preventDefault();
+      }
     });
-});
```

Comment:
Please add this line back. It breaks the JS. If you tested this code, this change should have broken it